src/pageServices/contentScripts/contextmenu.js

```diff
@@ -174,9 +174,9 @@ export const runContextMenu = () => {
       const mouseOffset = parseInt(ContextUtil.getProperty(options, "mouse_offset", 2));
 
       if (windowWidth - clickCoordsX < menuWidth) {
-        menu.style.left = windowWidth - menuWidth + pageXOffset + "px";
+        menu.style.left = windowWidth - menuWidth + scrollX - 13 + "px";
```

Comment:
lets create a constant for `13` magic value

Reply:
done